Common/UnitDefinitions/MassFraction.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Name": "MassFraction",
```

Comment:
I'm a little unsure about this one. Generally, you should be able to use and extend [Ratio](https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/Ratio.json) to represent these units.

The reason I am hesitant is that there are probably infinite variations of units that are dimensionless and that would have their own unique unit abbreviations. Is it possible that you can do with using Ratio? Or is there a very compelling usecase for adding this that you can't get with Ratio? I'd like to learn more.

Reply:
The first reason for including both entities & their respective units (like g/kg) is for parsing: I have "documents" that contain "concentrations" using any of the measure units in the list. Having the possibility to easily disambiguate the user input in a given context (i.e. chem. solutions) is key for both IR and HCI.
The second reason (and the actual reason I did not do separate PRs for each type) is the intricate way that the transformation operations bind Conc. Units < - Mass | Density | AmountOfSubstance MolarMass | Volume -> Conc. Units. For example MassConcentration = VolumeConcentration  * Density. Such operations wouldn't be possible with simple Ratios in the middle of it all.  Also a Ratio, IMAHO, is no more 'type-safe' than a Double. I've thought a little about how generics could help with the Ratios/Fractions but ended up with nothing useful- except maybe that you _have_ to know what you're referring to and not only in terms of units. A VolumeConcentration is the "Ratio< Volume >" between "constituent" / "mixture",  and that should not be "assignable from" any Ratio< Volume >, let alone Ratio. 
As a side note- I haven't implemented the previously mentioned transformation-operations for the MassFraction unit as they would have required working with lists of constituents. However, the transformations do become trivial if you're working in the getter/setter/function of a "Solute" (constituent) that is part of a MultiComponentSolution (mixture).

Reply:
Thanks for the detailed clarification. I don't yet fully grasp all the domain concepts here, but I think I get the gist of it. If I am reading it right, you are looking for two benefits:

1. Parsing strings like "1 g/kg"
2. Type safety and result type inference for calculations

I think these are valid reasons to have its own type.